docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,572 @@
+interact with your tables via Flink SQL. Here is an example using [SQLLine](https://julianhyde.github.io/sqlline/manual.html). 
+
+```shell
+sqlline> !connect jdbc:flink://localhost:8083
+```
+
+```shell
+sqlline version 1.12.0
+sqlline> !connect jdbc:flink://localhost:8083
+Enter username for jdbc:flink://localhost:8083:
+Enter password for jdbc:flink://localhost:8083:
+0: jdbc:flink://localhost:8083> CREATE TABLE T(
+. . . . . . . . . . . . . . .)>      a INT,
+. . . . . . . . . . . . . . .)>      b VARCHAR(10)
+. . . . . . . . . . . . . . .)>  ) WITH (
+. . . . . . . . . . . . . . .)>      'connector' = 'filesystem',
+. . . . . . . . . . . . . . .)>      'path' = 'file:///tmp/T.csv',
+. . . . . . . . . . . . . . .)>      'format' = 'csv'
+. . . . . . . . . . . . . . .)>  );
+No rows affected (0.122 seconds)
+0: jdbc:flink://localhost:8083> INSERT INTO T VALUES (1, 'Hi'), (2, 'Hello');
++----------------------------------+
+|              job id              |
++----------------------------------+
+| fbade1ab4450fc57ebd5269fdf60dcfd |
++----------------------------------+
+1 row selected (1.282 seconds)
+0: jdbc:flink://localhost:8083> SELECT * FROM T;
++---+-------+
+| a |   b   |
++---+-------+
+| 1 | Hi    |
+| 2 | Hello |
++---+-------+
+2 rows selected (1.955 seconds)
+0: jdbc:flink://localhost:8083>
+```
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/jdbcdriver/) 
+* [FLIP-293: Introduce Flink Jdbc Driver For Sql Gateway](https://cwiki.apache.org/confluence/display/FLINK/FLIP-293%3A+Introduce+Flink+Jdbc+Driver+For+Sql+Gateway)
+
+
+### Stored Procedure Support for Flink Connectors
+
+Stored procedures have been an indispensable tool in traditional databases,
+offering a convenient way to encapsulate complex logic for data manipulation
+and administrative tasks. They also offer the potential for enhanced
+performance, since they can trigger the handling of data operations directly
+within an external database. Other popular data systems like Trino and Iceberg
+automate and simplify common maintenance tasks into small sets of procedures,
+which greatly reduces users' administrative burden.
+
+This new update primarily targets developers of Flink connectors, who can now
+predefine custom stored procedures into connectors via the Catalog interface.
+The primary benefit to users is that connector-specific tasks that previously
+may have required writing custom Flink code can now be replaced with simple
+calls that encapsulate, standardize, and potentially optimize the underlying
+operations. Users can execute procedures using the familiar `CALL` syntax, and
+discover a connector's available procedures with `SHOW PROCEDURES`. Stored
+procedures within connectors improves the extensibility of Flink's SQL and
+Table APIs, and should unlock smoother data access and management for users.
```

Comment:
Is it possible to add an example in here?
"
Users can use the Call statement to call the built-in stored procedures provide by Catalog directly (Note: For the built-in  stored procedures in Catalog, please refer to the documentation of the corresponding Catalog).
For example, you can use the following Call statement to perform a compaction operation on a [Paimon](https://paimon.apache.org/) table.
```sql
CREATE TABLE `paimon`.`default`.`T` (
  id BIGINT PRIMARY KEY NOT ENFORCED,
  dt STRING, -- format 'yyyy-MM-dd'
  v STRING
);

-- use catalog before call procedures
USE CATALOG `paimon`;

-- compact the whole table using call statement
CALL sys.compact('default.T');
```
"